### PR TITLE
feat(app): add robotName confirmation screen

### PR DESCRIPTION
**Pull request by koji**
# Overview
Add robotName confirmation screen
Close RCORE-385
Screenshot 2022-12-08 221604

design
https://www.figma.com/file/AoTLAYuWawlaWItB1umOjr/OT-3-On-Device-Display?node-id=1776%3A30238&t=ECy6VKgxraam41T1-0

# Changelog
- add robotName confirmation screen and test

# Review requests
- Run `make -C app dev OT_APP_IS_ON_DEVICE=1` or push this branch to odd then go to `To Robot Dashboard` and click `Name confirmation` 

# Risk assessment
low

### CODE REVIEW

File: app/src/pages/OnDeviceDisplay/ConfirmRobotName.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            marginTop={SPACING.spacingXXL}
+            fontSize="1.625rem"
+            lineHeight="2.1875rem"
+            fontWeight={TYPOGRAPHY.fontWeightRegular}
+          >
+            {t('your_robot_is_ready_to_go')}
+          </StyledText>
+          <PrimaryButton
+            marginTop={SPACING.spacingXXL}
+            onClick={() => console.log('not designed yet')}
```
Comment (brenthagen) [inline app/src/pages/OnDeviceDisplay/ConfirmRobotName.tsx]: maybe link to robot dashboard?

**Review by brenthagen — APPROVED** [review]
looks like two versions of the image were added, probably `app/src/assets/odd_abstract@x2.png` shouldn't be there

File: app/src/pages/OnDeviceDisplay/ConfirmRobotName.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { PrimaryButton } from '../../atoms/buttons'
+import screenImage from '../../assets/images/odd/odd_abstract@x2.png'
+
+const IMAGE_ALT = 'finish setting up a robot'
+
+export function ConfirmRobotName(): JSX.Element {
+  const { t } = useTranslation('device_settings')
+  const localRobot = useSelector(getLocalRobot)
+  const robotName = localRobot?.name
+  return (
```
Comment (brenthagen) [inline app/src/pages/OnDeviceDisplay/ConfirmRobotName.tsx:29]: also, should this page have the StepMeter?

Comment (koji) [reply app/src/pages/OnDeviceDisplay/ConfirmRobotName.tsx:29]: oh, yes.
I will add StepMeter.

**Comment by koji** [comment]
- [x] address comments

**Pull request merged**